ocs_ci/ocs/resources/mcg.py

```diff
@@ -102,16 +103,28 @@ def __init__(self):
 
         self.s3_client = self.s3_resource.meta.client
 
-        # Give NooBaa's ServiceAccount permissions in order to execute CLI commands
-        registry.add_role_to_user(
-            'cluster-admin', constants.NOOBAA_SERVICE_ACCOUNT,
-            cluster_role=True
-        )
-
         self.operator_pod = Pod(
-            **get_pods_having_label(constants.NOOBAA_OPERATOR_POD_LABEL, self.namespace)[0]
+            **get_pods_having_label(
+                constants.NOOBAA_OPERATOR_POD_LABEL, self.namespace
+            )[0]
         )
 
+        # TODO: Add version verification to see if an additional cp isn't needed
+        # If the MCG CLI binary isn't found, copy it from the operator pod
+        if not os.path.isfile(constants.NOOBAA_OPERATOR_LOCAL_CLI_PATH):
+            cmd = (
```

Comment:
@MeridianExplorer can we use `oc cp` or `oc rsync` <- for that I know we already have function for rsync.

https://www.mankier.com/1/oc-cp
https://www.mankier.com/1/oc-rsync

Reply:
Nope, sadly, we can't use either. The pod doesn't contain `tar`, nor `rsync`. There was a long discussion about this in Slack

Reply:
Also, I had to work around `shlex` splitting the command in a way that wouldn't let it run in the way I wanted it to, thus the separate use in `subprocess_run` instead of `exec_cmd`

Reply:
+1 on using rsync and fixing pod in future, looks clean.

Reply:
I believe that'd up to @nimrod-becker and the team - they're the owners of the image, so it's up to them to include tar/rsync in it, if they choose :)
I'd definitely support it.

Reply:
ok, thanks for info. I thought that this is the case as I experienced missing tar/rsync in some of the pod in the past as well.

 Thanks